tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -242,7 +253,11 @@ def test_get_children_tags_invalid_taxonomy(self) -> None:
         """
         Calling get_children_tags on free text taxonomies gives an error.
         """
-        free_text_taxonomy = Taxonomy.objects.create(allow_free_text=True, name="FreeText")
+        free_text_taxonomy = Taxonomy.objects.create(
```

Comment:
nit: this is a good opportunity to change these `Taxonomy.object.create`s to `tagging_api.create_taxonomy`.

Reply:
Yep :+1:

Reply:
Updated here: https://github.com/openedx/openedx-learning/pull/145/commits/f53025c0f6868443006abdc7a55ccd2a65149d1a